<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Symfony\Component\Routing\Exception\ResourceNotFoundException" namespace="Symfony\Component\Routing\Exception" name="ResourceNotFoundException" abstract="false" final="false" start="21" end="23">
  <file path="/home/web/example/src/freegame_developer/vendor/symfony/routing/Exception" file="ResourceNotFoundException.php" realpath="/home/web/example/src/freegame_developer/vendor/symfony/routing/Exception/ResourceNotFoundException.php" size="548" time="2016-08-16T10:55:04+00:00" unixtime="1471344904" sha1="bced4b5181142dbc343411375bf97bd42465fd7a" relative="freegame_developer/vendor/symfony/routing/Exception/ResourceNotFoundException.php"/>
  <docblock>
    <description compact="The resource was not found.">This exception should trigger an HTTP 404 response in your application code.</description>
    <author value="Kris Wallsmith &lt;kris@symfony.com&gt;"/>
  </docblock>
  <extends full="RuntimeException" namespace="" name="RuntimeException"/>
  <implements full="Symfony\Component\Routing\Exception\ExceptionInterface" namespace="Symfony\Component\Routing\Exception" name="ExceptionInterface"/>
</class>
